api/cpp/include/slint_window.h

```diff
@@ -105,12 +105,14 @@ class WindowAdapterRc
         slint_windowrc_set_component(&inner, &item_tree_rc);
     }
 
-    template<typename Component, typename Parent>
-    void show_popup(const Parent *parent_component, cbindgen_private::Point p, bool close_on_click,
-                    cbindgen_private::ItemRc parent_item) const
-    {
-        auto popup = Component::create(parent_component).into_dyn();
-        cbindgen_private::slint_windowrc_show_popup(&inner, &popup, p, close_on_click,
+    template<typename Component, typename Parent, typename XGetter, typename YGetter>
+    void show_popup(const Parent *parent_component, XGetter x_getter, YGetter y_getter,
```

Comment:
can be done later, but i'd make only one getter that return a point.

Reply:
Ok, want to do it as part of the position refactoring or shall I do it as follow-up?

Reply:
i can do it

Reply:
Thanks :)